io/router/transfer.go

```diff
@@ -0,0 +1,45 @@
+package router
+
+import (
+	"gioui.org/io/transfer"
+	"strings"
+)
+
+type transferQueue struct {
+	initialized   bool
+	schemes       []transfer.SchemeOp
+	firstURLEvent transfer.URLEvent
+}
+
+func (q *transferQueue) Push(evt transfer.URLEvent, events *handlerEvents) {
+	if !q.initialized && q.firstURLEvent.URL == nil {
```

Comment:
Why not drop `q.initialized` and replace every check with `q.firstURLEvent.URL == nil`?